.github/workflows/flownet.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           TESTDATA_REPO_OWNER: equinor
           # If you want the CI to (temporarily) run against another branch than master,
           # change the value her from "master" to the relevant branch name.
-          TESTDATA_REPO_BRANCH: master
+          TESTDATA_REPO_BRANCH: observation-errors-in-config-yaml
```

Comment:
Merge https://github.com/equinor/flownet-testdata/compare/observation-errors-in-config-yaml and put back to master before merging.